container/ubuntu16_04/layers/bazel/version.bzl

```diff
@@ -0,0 +1,2 @@
+BAZEL_VERSION = "0.15.0"
```

Comment:
lets keep dict from version to sha so that we can keep here multiple versions/shas?

Reply:
Done